pkg/core/blockchain.go

```diff
@@ -1670,6 +1672,7 @@ func (bc *Blockchain) initVerificationVM(ic *interop.Context, hash util.Uint160,
 		}
 		initMD := cs.Manifest.ABI.GetMethod(manifest.MethodInit)
 		v.LoadScriptWithHash(cs.NEF.Script, hash, callflag.ReadStates)
+		v.Context().NEF = &cs.NEF
```

Comment:
You need to also set `LoadToken` here.

Reply:
Fixed (set it when spawn in `verifyHashAgainstScript`.